xclim/indices/_multivariate.py

```diff
@@ -153,18 +154,34 @@ def cold_and_dry_days(tas, tgin25, pr, wet25, freq="YS"):
     # return c.resample(time=freq).sum(dim='time')
 
 
-@declare_units("days", tasmax="[temperature]", tasmin="[temperature]")
-def daily_freezethaw_cycles(tasmax, tasmin, freq="YS"):
+@declare_units(
+    "days",
+    tasmax="[temperature]",
+    tasmin="[temperature]",
+    thresh_tasmax="[temperature]",
+    thresh_tasmin="[temperature]",
+)
+def daily_freezethaw_cycles(
+    tasmax,
+    tasmin,
+    thresh_tasmax="UNSET 0 degC",
```

Comment:
What purpose serves this UNSET ?

Reply:
It's a trick I picked up to warn users of a future deprecation. 

If they don't set the thresh_tasmax or thresh_tasmin manually, this creates a very visible warning that in the future they will need to do so. We are eventually going to strictly integrate the `*` into function calls so that call signatures cannot be performed based on order. This relates to the discussion brought up in #257.